pkg/controller/endpointslice/endpointslice_controller.go

```diff
@@ -159,13 +165,13 @@ func NewController(ctx context.Context, podInformer coreinformers.PodInformer,
 	if utilfeature.DefaultFeatureGate.Enabled(features.TopologyAwareHints) {
 		nodeInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
 			AddFunc: func(obj interface{}) {
```

Comment:
Nit: s/`obj`/`_` given we don't plug the `obj` down the call hierarchy.

Ditto for `DeleteFunc`.

Reply:
Done